Dockerfile

```diff
@@ -105,7 +105,7 @@ RUN <<EOF
         -DgRPC_BUILD_TESTS=OFF \
         -DgRPC_SSL_PROVIDER=package \
         ../..
-    make -j4 install
+    make -j8 install
```

Comment:
In case 8 is a bit too agressive on some systems I suggest adding a build arg at the top of the Dockerfile e.g 
`ARG NO_COMPILE_JOBS=8` and then 
'make -j"$NO_COMPILE_JOBS"' in usage